tensorflow/core/api_def/base_api/api_def_Substr.pbtxt

```diff
@@ -42,6 +42,8 @@ For each string in the input `Tensor`, creates a substring starting at index
 If `len` defines a substring that would extend beyond the length of the input
 string, then as many characters as possible are used.
```

Comment:
Maybe instead of adding another paragraph, change this sentence to:

If 'len' defines a substring that would extend beyond the length of the input string, or if 'len' is negative, tten as many characters as possible are used.